.gitlab-ci.yml

```
@@ -490,6 +492,8 @@ trigger:backend-integration-testing:mender-qa:
 
 .template:trigger:acceptance-testing:
   stage: test
+  rules:
+    - if: $CI_COMMIT_BRANCH != "master"
```

```suggestion
    - if: "$CI_COMMIT_BRANCH" != "master"
```
I always wonder does it change anything in the gitlab lingo.

---

Thanks for your eye on better safe than sorry terrain. However,

This variable is guaranteed from GitLab to exist. And the issue with quoting `$CI_COMMIT_BRANCH`  it is that then for the line to be valid markdown we need to quote also the whole expression, which IMO degrades readability and in this case is not worth it:
```suggestion
    - if: '"$CI_COMMIT_BRANCH" != "master"'
```